tests.yaml

```diff
@@ -773,10 +773,10 @@ tests:
     values: '1+0x5'
   promql_expr_test:
   - eval_time: 5m
-    expr: node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate
+    expr: node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate
```

Comment:
We would need tests for both rules now:

- `node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate`
- `node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate`